R/callOpenTiles.R

```diff
@@ -420,6 +435,11 @@ setMethod(
       )
 
       summarizedData <- as.data.frame(summarizedData)
+        
+      #Remove any columns with missing labels
+      cellTypeList <- summarizedData[[cellPopLabel]]
+      summarizedData <- summarizedData[!is.na(cellTypeList),,drop=FALSE]
+      cellTypeList <- cellTypeList[!is.na(cellTypeList)]
```

Comment:
we don't need this line (442) anymore since cellTypeList is not used, right?

Reply:
Good point. Yes, it's not needed .